index.js

```diff
@@ -249,6 +249,19 @@ function bootstrap(options) {
       return next();
     });
   }
+
+  app.get('/session-timeout', (req, res, next) => {
+    if (req.cookies['hof-wizard-sc'] && req.session.exists !== true) {
+      const err = new Error('Session expired');
+      err.code = 'SESSION_TIMEOUT';
+      return next(err);
+    }
+    const err = new Error('Not Found');
+    err.status = 404;
+    const locals = Object.assign({}, req.translate('errors'));
+    return res.status(404).render('404', locals['404']);
```

Comment:
It might be worth checking if locals['404'] exists before using it.  By adding this code, I think it stops a blank page if translation data is missing.
```suggestion
  const locals = Object.assign({}, req.translate('errors'));
  if (locals && locals['404']) {
    return res.status(404).render('404', locals['404']);
  } else {
    // Fallback: render a basic 404 page if translation is missing
    return res.status(404).send('Page Not Found');
  }
```